vertx-pg-client/src/test/java/io/vertx/pgclient/impl/codec/ResponseHelper.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.pgclient.impl.codec;
+
+import io.vertx.pgclient.PgException;
+
+public final class ResponseHelper {
```

Comment:
rather reuse the existing `io.vertx.pgclient.impl.util.Util` class that contains already similar static methods

Reply:
The Response methods setMessage, setSeverity, etc. are package-private.
Response is in main/java/io/vertx/pgclient/impl/codec/Response.java.
ResponseHelper is a test helper located in test/java/io/vertx/pgclient/impl/codec/ResponseHelper.java. It is in test/... because it is needed for testing only. It is in io/vertx/pgclient/impl/codec to have access to the package-private methods of Response.java.
Moving the ResponseHelper methods into main/io/vertx/pgclient/impl/Util.java fails because of the package-private access restriction, and having a test helper needed for testing only should not got into main/...